### PR TITLE
Show the block appender even if the last block is non empty paragraph

### PR DESCRIPTION
**Pull request by youknowriad**
Everything is in the title.

**Testing instructions**

 - Write some text
 - Ensure you can click under the last paragraph and continue typing
 - Ensure that there's no tab trap

### CODE REVIEW

File: editor/components/default-block-appender/index.js [hunk 1]
```diff
@@ -45,10 +44,10 @@ export default compose(
 		( state, ownProps ) => {
 			const isEmpty = ! getBlockCount( state, ownProps.rootUID );
 			const lastBlock = getBlock( state, ownProps.lastBlockUID );
-			const isLastBlockDefault = get( lastBlock, 'name' ) === getDefaultBlockName();
```
Comment (youknowriad) [inline editor/components/default-block-appender/index.js]: This was not possible originally because we didn't have the `isUnmodifiedDefaultBlock` function

**Comment by jasmussen** [comment]
Getting this error, despite having done `npm intall` first. I also tried `npm update`:

```
react-dom.min.3583f8be.js:162 TypeError: Cannot read property 'name' of null
    at isUnmodifiedDefaultBlock (utils.js:23)
    at Function.mapToProps (index.js:47)
    at mapToPropsProxy (wrapMapToProps.js:43)
    at Function.detectFactoryAndVerify (wrapMapToProps.js:52)
    at mapToPropsProxy (wrapMapToProps.js:43)
    at handleFirstCall (selectorFactory.js:26)
    at pureFinalPropsSelector (selectorFactory.js:74)
    at Object.runComponentSelector [as run] (connectAdvanced.js:26)
    at Connect.initSelector (connectAdvanced.js:178)
    at new Connect (connectAdvanced.js:119)
```

Any idea what I'm doing wrong?

**Comment by youknowriad** [comment]
@jasmussen my bad I forgot to test on empty posts, it should be fixed now :)

**Comment by jasmussen** [comment]
This is working amazingly well! 👍 👍

**Review by aduth — APPROVED** [review]
👍

**Pull request merged**